src/mscorlib/src/System/Threading/Tasks/TaskContinuation.cs

```diff
@@ -803,7 +803,7 @@ protected static void ThrowAsyncIfNecessary(Exception exc)
 #endif // FEATURE_COMINTEROP
                 {
                     var edi = ExceptionDispatchInfo.Capture(exc);
-                    ThreadPool.QueueUserWorkItem(s => ((ExceptionDispatchInfo)s).Throw(), edi);
+                    ThreadPool.QueueUserWorkItem(s => ((ExceptionDispatchInfo)s).Throw(), edi, preferLocal: true);
```

Comment:
As with the earlier case, we're about to crash.  Why does contention here matter?